artifactory.go

```diff
@@ -347,6 +346,44 @@ func (b *backend) getRootCert(config adminConfiguration) (cert *x509.Certificate
 	return
 }
 
+type Feature struct {
+	FeatureId string `json:"featureId"`
+}
+
+type Usage struct {
+	ProductId string    `json:"productId"`
+	Features  []Feature `json:"features"`
+}
+
+func (b *backend) callHome(config adminConfiguration, featureId string) {
+	b.Backend.Logger().Debug("callHome")
```

Comment:
This should probably have the featureId string?
```suggestion
	b.Backend.Logger().Debug("callHome", "featureId", featureId)
```

Reply:
Actually I meant to remove this before opening the PR.